synthesizer/src/program/mapping/value/parse.rs

```diff
@@ -17,7 +17,7 @@
 use super::*;
 
 impl<N: Network> Parser for MapValue<N> {
-    /// Parses a string into the value statement of the form `value {name} as {register_type};`.
+    /// Parses a string into the value statement of the form `value {name} as {plaintext_type}.public;`.
```

Comment:
Some files say `{plaintext_type}.public` and most files say `{register_type}` now. We should be consistent.